conductor/src/metrics_reporter.rs

```diff
@@ -42,7 +42,7 @@ pub async fn run_metrics_reporter() -> Result<()> {
         env::var("METRICS_EVENTS_QUEUE").expect("METRICS_EVENTS_QUEUE must be set");
 
     queue.init().await?;
-    queue.create(&metrics_events_queue).await?;
+    queue.create_partitioned(&metrics_events_queue).await?;
```

Comment:
@ChuckHend just in case there's a reason we can't delete this, updating it to `create_partitioned` _should_ resolve the issue. Going to try with a branch based image

Reply:
First installation with this change looks good. I am going to run through a few more.